tests/MassTransit.MartenIntegration.Tests/SagaPersistenceTests.cs

```diff
@@ -74,7 +74,7 @@ public async Task An_observed_message_should_find_and_update_the_correct_saga()
         [OneTimeSetUp]
         public async Task OneTime()
         {
-            await _store.Schema.ApplyAllConfiguredChangesToDatabaseAsync();
+            await _store.Storage.ApplyAllConfiguredChangesToDatabaseAsync();
         }
```

Comment:
I haven't added a conditionally to this, as the unit test project is explicitly targeting net7